README.md

```diff
@@ -31,6 +31,8 @@ Fork Client is a fork of the official Telegram for Android application.
 - Added upload date for profile photos.
 - Added ability to see the profile info from the dialogs list via context menu.
 - Added ability to see unread count when you want to mark as read multiple dialogs.
+- Option to directly open the archive on pulldown
+- PiP mode for YouTube's in-app player
```

Comment:
I'm not sure that's a feature. This function was always, just for some reason it was disabled by default in the latest official version.

Reply:
It was disasbled because goes against Google's ToS, because it's now considered a premium feature in their app - so they will remove from their store any app allowing to PiP YouTube videos. Since Telegram is not allowed to use it anymore, and since Forkgram is not distributed via the PlayStore, I think we can consider this a feature Forkgram has over the normal app